weather-berlin/app.js

```diff
@@ -15,24 +28,9 @@ fetch(url)
         getElement('temperature').innerText = data['current_weather']['temperature']
         getElement('windspeed').innerText = data['current_weather']['windspeed']
         getElement('weathercode').innerText = data['current_weather']['weathercode']
-        getElement('winddirection').innerText = data['current_weather']['winddirection']        })
+        getElement('winddirection').innerText = data['current_weather']['winddirection']
+        message(data['current_weather']['temperature'])
```

Comment:
@ecemonkol this is a better way to display messages. there are some reasons

1. We may use the message in somewhere different. For example in page title. In order to avoid code-repeat, we keep the logic in one place.
2. You wanted to display temperature message while the URL was fetched. The logic was executed before the fetch action is done.
3. now whenever it fetches the URL successfully, it runs the new function.